<p><em><em>Another guest post by <a href="https://web.archive.org/web/20120611031744/http://network.nature.com/people/alexander-knoll/profile" data-cite-meta="{}">Alex Knoll <span class="citation" data-cites="https://web.archive.org/web/20120611031744/http://network.nature.com/people/alexander-knoll/profile">(<strong>https://web.archive.org/web/20120611031744/http://network.nature.com/people/alexander-knoll/profile?</strong>)</span></a> reporting from the German Genetics Society Meeting in Cologne.</em></em></p>
<p>Just after lunch the next decision: Neurogenetics or Evolution? Here I went with the topic that is less far from my field, and attended the evolution session. Welcome to the modern times, because next generation sequencing and high throughput are the way to go!</p>
<p>These were the methods of choice for Julia Zeitlinger from the Stowers Institute for Medical Research in Kansas City to compare transcription factor binding sites between several Drosophila species. This goes back to the question, which process contributes more to evolution: mutations in the protein-coding parts of genes, or the sequences needed for the transcriptional regulation of these genes, the so-called cis-regulatory elements. There is evidence in some organisms of a high turnover of transcription factor binding sites, and Julia Zeitlinger wanted to compare these in the genus Drosophila. This was done with chromatin immunoprecipitation approaches, where you analyze the sequence of DNA fragments to which a target protein like a transcription factor is bound. Earlier, this was followed by hybidisation of microarrays (ChIP-chip), today these DNA fragments can be directly sequenced with next generation methods (ChIP-seq). They chose the dorso-ventral patterning in the Drosophila embryo as a model system, with transcription factors like Twist that are well studied, and that have conserved binding sites. Surprisingly, they found that Twist binds to about 8 times more positions than were expected; sometimes Twist bound at more than one position at target genes.</p>
<p>Across Drosophila species the binding sites of Twist were very similar, especially near transcriptional start sites the positions are mostly conserved. Although movements of binding sites exist, they are rare. So it seems that there is evolutionary pressure to maintain the sites.</p>
<p>Also about the evolutionary contributions of cis-regulatory regions was the talk by Diethard Tautz from the Max Planck Institute for Evolutionary Biology in Plants, Germany. But he addressed them with population genetics of wild house mouse strains. With microarray and qPCR experiments, his lab was able to find that the majority of genes they looked at follow a neutral evolution. When this is not the case, most of those genes show only tissue-specific changes in expression, which points to cis-regulatory elements.</p>
<p>They also found a newly evolved gene in the mouse: In a region that is completely free of annotated transcripts in other mammals, one transcript can be found in the mouse. Poldi, as it is called, is expressed in the mouse testis and encodes a non-protein coding RNA. They were able to follow its birth in the genus Mus, where it appeared about 2.5 million years ago by changes in promoter regions. And although Poldi appeared relatively recently, it plays a role in the mouse. A knockout results in mice with reduced sperm motility and smaller testis, because several chromatin-regulating proteins show different expression levels in the absence of Poldi.</p>
<p>Now on to some computational biology by Andrew Hufton from the Max Planck Institute for Molecular Genetics in Berlin. He analyzed a special kind of cis-regulatory elements: the so-called conserved non-coding elements (CNEs), that don't show much chance even over great evolutionary distances. Why do they exist? There are two competing hypotheses; either the CNEs repress genome rearrangements, or they promote the retention of duplicate genes nearby. To test both hypotheses, Andrew Hufton identified so-called phylogenetically conserved non-coding elements. These are elements associated with gene families and are not biased for one of the hypotheses. With experiments in zebrafish, he found that most of the PCNEs are enhancers, sequences involved in the regulation of gene expression.</p>
<p>Concerning the two competing hypotheses, he found a strong association between PCNE and synteny conservation – the first hypothesis: the PCNEs are enriched around genes with an ancient gene order. For the retention of duplicated genes, he showed us an alternative model.</p>
<p>In the last short talk of this session, Stephan Greiner told us about a plant with a strange inheritance. At the Max Planck Institute for Molecular Plant Physiology at Potsdam-Golm in Germany, he is working on the Evening Primrose Oenothera. It combines some very interesting genetic features like easy crossings between species, fertility of the resulting hybrids, biparental transmission of chloroplasts (that are usually inherited maternally) and something very strange called permanent-translocation heterozygosity: The chromosomes of Oenothera easily break at the centromeres, so that translocations of chromosome arms happen. This leads to rings or chains of chromosomes during meiosis that suppress homologous recombination and the intermixing of haploid chromosomes. In the end, there are a handful of stable genomes that a transmitted stably, and a hybrid plant with a combination of two of these genomes will always have hybrid offspring.<br />
Add in the possibility to exchange one of several different chloroplast genomes in only two generations, and you will be able to generate compatibility charts of nuclear and chloroplast genomes, where lethal combinations and less severe phenotypes occur. Taken one step further, it is possible to identify speciation genes in the chloroplast genome.</p>
<div id="refs" class="references csl-bib-body hanging-indent" data-line-spacing="2" role="doc-bibliography">
<div id="ref-https://web.archive.org/web/20151003101404/http://www.bl.uk/aboutus/stratpolprog/legaldep/" class="csl-entry" role="doc-biblioentry">
(n.d.-a). In <em>web.archive.org</em>. Retrieved February 21, 2021, from <a href="https://web.archive.org/web/20150923191606/http://www.bl.uk/aboutus/stratpolprog/legaldep/">https://web.archive.org/web/20150923191606/http://www.bl.uk/aboutus/stratpolprog/legaldep/</a>
</div>
<div id="ref-https://web.archive.org/web/20161010225753/http://retractionwatch.wordpress.com/2012/01/30/an-arxiv-for-all-of-science-f1000-launches-new-immediate-publication-journal/" class="csl-entry" role="doc-biblioentry">
(n.d.-b). In <em>web.archive.org</em>. Retrieved February 21, 2021, from <a href="https://web.archive.org/web/20170831024033/https://retractionwatch.wordpress.com/2012/01/30/an-arxiv-for-all-of-science-f1000-launches-new-immediate-publication-journal/">https://web.archive.org/web/20170831024033/https://retractionwatch.wordpress.com/2012/01/30/an-arxiv-for-all-of-science-f1000-launches-new-immediate-publication-journal/</a>
</div>
<div id="ref-https://web.archive.org/web/20161010225753/http://arxiv.org/" class="csl-entry" role="doc-biblioentry">
arXiv.org e-Print archive. (2016). In <em>web.archive.org</em>. <a href="https://web.archive.org/web/20161013002949/https://arxiv.org/">https://web.archive.org/web/20161013002949/https://arxiv.org/</a>
</div>
<div id="ref-https://web.archive.org/web/20161010225753/http://blogs.nature.com/news/2012/01/f1000-launches-fast-open-science-publishing-for-biology-and-medicine.html" class="csl-entry" role="doc-biblioentry">
F1000 launches fast, open science publishing for biology and medicine : News blog. (2016). In <em>web.archive.org</em>. <a href="https://web.archive.org/web/20161004092538/http://blogs.nature.com/news/2012/01/f1000-launches-fast-open-science-publishing-for-biology-and-medicine.html">https://web.archive.org/web/20161004092538/http://blogs.nature.com/news/2012/01/f1000-launches-fast-open-science-publishing-for-biology-and-medicine.html</a>
</div>
<div id="ref-https://web.archive.org/web/20161010225753/http://precedings.nature.com/" class="csl-entry" role="doc-biblioentry">
Home : Nature Precedings. (2016). In <em>web.archive.org</em>. <a href="https://web.archive.org/web/20161012221933/http://precedings.nature.com/">https://web.archive.org/web/20161012221933/http://precedings.nature.com/</a>
</div>
<div id="ref-https://web.archive.org/web/20161010225753/http://blogs.plos.org/mfenner/2010/10/16/in-which-i-suggest-a-preprint-archive-for-clinical-trials/" class="csl-entry" role="doc-biblioentry">
In which I suggest a preprint archive for clinical trials Gobbledygook. (2016). In <em>web.archive.org</em>. <a href="https://web.archive.org/web/20161019024534/http://blogs.plos.org/mfenner/2010/10/16/in-which-i-suggest-a-preprint-archive-for-clinical-trials/">https://web.archive.org/web/20161019024534/http://blogs.plos.org/mfenner/2010/10/16/in-which-i-suggest-a-preprint-archive-for-clinical-trials/</a>
</div>
<div id="ref-https://web.archive.org/web/20161010225753/https://twitter.com/#!/F1000Research" class="csl-entry" role="doc-biblioentry">
Twitter. It’s what’s happening. (2016). In <em>web.archive.org</em>. <a href="https://web.archive.org/web/20161010225105/https://twitter.com/">https://web.archive.org/web/20161010225105/https://twitter.com/</a>
</div>
</div>
