<p><em><em>Another guest post by <a href="https://web.archive.org/web/20120611033341/http://network.nature.com/people/alexander-knoll/profile">Alex Knoll</a> reporting from the German Genetics Society Meeting in Cologne.</em></em></p>
<p>The last session on Thursday was split into two talks by Philippe Sansonetti from the Institute Pasteur in Paris and Julian Parkhill from the Sanger Institute in Hinxton, Great Britain. Both talked about pathogenic microbes, but from a different perspective.</p>
<p>After Philippe Sansonetti told us about <em><em>Shigella</em></em> and the innate immune system during invasion of the gut epithelium, Julian Parkhill gave us some insights into two (genomically) very different organisms. <em><em>Salmonella typhi</em></em>, responsible for typhoid fever, has evolved relatively recently by changing its niche; it went from being a gut pathogen to a systemic pathogen. This can be seen in its genome, because its evolution is not characterized by the gain of genes, but actually by the loss of many genes through single mutations. For bacteria, <em><em>Salmonella typhi</em></em> is rather strange: There is almost no recombination between strains, so there are very few SNPs that can by typed. By looking at 200 genes in about 200 strains, they found 88 SNPs in total! Evolutionarily, their data point to mostly long-term neutral mutations, with only very few genes to be found that show evidence of positive selection. But the SNP data they acquired could be put to good use nonetheless: They were able to do an epidemiological analysis in the field by collecting S. typhi strains around Kathmandu.</p>
<p>The opposite of <em><em>S. typhi</em></em> in terms of recombination and SNPs is <em><em>Staphylococcus aureus</em></em>, which comprises a hugely diverse group of freely recombining strains. With next generation sequencing, they were able to track the appearance and travels of a specific strain around the world. They even got enough SNPs in their dataset to be able to distinguish strains from different wards in a single hospital!<br />
The talk was also a lesson in technological progress: While the sequencing of the <em><em>Salmonella typhi</em></em> genome a few years ago took 1.5 years, the next generation sequencing of <em><em>Staphylococcus aureus</em></em> allowed the analysis of 66 genomes per run and week. Finally, we had a glimpse into the future: They are getting into single cell genome sequencing by doing laser capture microsdissection, and are already able to reach about 98% coverage!</p>
<p>Following this last talk of the day, the German Genetics Society awarded the honorary membership to Rolf Knippers. At least German speaking biologists should know him by name because of his well-known (and great, in my opinion) textbook <em><em>Molekulare Genetik</em></em> (Molecular Genetics), which first appeared in the early 1970s. But that was not the only reason for the award. Actually, Rolf Knippers has been a very good researcher and teacher, too! I didn't know him apart from his textbook before, but the laudatio and his acceptance speech (which he started by humbly asking if he deserved the award) made me wish I had met him earlier, perhaps as a student or even as a member of his lab.</p>
