<p><em><em>The final guest post by <a href="https://web.archive.org/web/20120611031645/http://network.nature.com/people/alexander-knoll/profile" data-cite-meta="{}">Alex Knoll <span class="citation" data-cites="https://web.archive.org/web/20120611031645/http://network.nature.com/people/alexander-knoll/profile">(<strong>https://web.archive.org/web/20120611031645/http://network.nature.com/people/alexander-knoll/profile?</strong>)</span></a> reporting from the German Genetics Society Meeting in Cologne.</em></em></p>
<h3 id="session-v">Session V</h3>
<p>Friday ended with two talks in session V, the first by Tony Hyman from the Max Planck Institute for Cell Biology and Genetics in Dresden. He looked at the ways that cells structure and organize their cytoplasm by comparison with non-biological systems.</p>
<p>Next up was a talk on the innate immune response against the influenza virus by University of Freiburg's Otto Haller. When viruses infect mammalian hosts, a battle between interferons and virulence factors starts to rage. The interferon-induced GTPase Mx1 is an important resistance factor against influenza A viruses in mice. Most lab mouse strains are natural knockouts for Mx1, so they needed to create congenic, wild-type Mx1-expressing mice to research its function.</p>
<p>Interestingly, Mx1 expression protects mice against the usually lethal 1918 Spanish flu influenza A, even at high doses. Mx proteins are highly conserved in mammalian species and belong to the dynamin superfamily of large GTPases. Otto Haller presented new and unpublished data (obtained in collaboration with Oliver Daumke's group at the Berlin Max Delbrück Center for Molecular Medicine) on the role of the structure of the human MxA GTPase in repressing the virus.</p>
<h3 id="session-vii">Session VII</h3>
<p>The final session of the meeting started with a talk by Andrew McMahon from the Harvard Stem Cell Institute in Cambridge, US, that I sadly missed. But one missed talk during the whole meeting is not that bad now, don't you think?<br />
Ueli Grossniklaus then talked about epigenetics in the struggle between the two parental genomes of an Arabidopsis embryo. At the University of Zürich in Switzerland, his group is looking at the timing of expression of paternal alleles and their phenotypic consequences, all controlled by the maternal genome.</p>
<p>The last talk was given by Bruce Beutler from the Scripps Research Institute in La Jolla, US. He is looking into the ways in which the mammalian immune system is able to recognize microbes as foreign, and then mount an adequate response. A first answer was found when the receptor for LPS, a component of all Gram-negative microbes, was identified by positional cloning as Tlr4, a Toll-like receptor. You perhaps know Toll as a developmental gene in the Drosophila embryo, but in adult flies, it is required for the immune response to fungal and bacterial infections. So perhaps other Toll-like receptors also recognize other microbial ligands? Yes, and Bruce Beutler's group has been looking into the signaling pathways that lead from recognition by a Toll-like receptor to the induction of cellular responses. He and his group have used a forward genetic screen of randomly mutagenized mice. To date, they have generated over 100,000 mutant lines, and identified 32 mutations affecting Toll-like receptor signaling. This allowed them to deduce biochemical pathways that mediate much of the innate immune response. A similar feat was done with the genes involved in the resistance against mouse cytomegalovirus, with susceptibility mutations found in sensing and signaling pathways, but also in homeostasis and in development.</p>
<div id="refs" class="references csl-bib-body hanging-indent" data-line-spacing="2" role="doc-bibliography">
<div id="ref-https://web.archive.org/web/20120610120821/http://www.crossref.org/help/Content/02_Getting_started/Displaying_DOIs_in_print_and_online.htm" class="csl-entry" role="doc-biblioentry">
CrossRef DOI Display Guidelines. (2012). In <em>web.archive.org</em>. <a href="https://web.archive.org/web/20120520234457/http://www.crossref.org/help/Content/02_Getting_started/Displaying_DOIs_in_print_and_online.htm">https://web.archive.org/web/20120520234457/http://www.crossref.org/help/Content/02_Getting_started/Displaying_DOIs_in_print_and_online.htm</a>
</div>
<div id="ref-https://web.archive.org/web/20101124191039/http://blogs.plos.org/mfenner/2010/09/07/orcid-as-unique-author-identifier-what-is-it-good-for-and-should-we-worry-or-be-happy/" class="csl-entry" role="doc-biblioentry">
ORCID as unique author identifier: what is it good for and should we worry or be happy? Gobbledygook. (2011). In <em>web.archive.org</em>. <a href="https://web.archive.org/web/20110116034110/http://blogs.plos.org/mfenner/2010/09/07/orcid-as-unique-author-identifier-what-is-it-good-for-and-should-we-worry-or-be-happy/">https://web.archive.org/web/20110116034110/http://blogs.plos.org/mfenner/2010/09/07/orcid-as-unique-author-identifier-what-is-it-good-for-and-should-we-worry-or-be-happy/</a>
</div>
<div id="ref-https://web.archive.org/web/20101124191039/http://www.ncbi.nlm.nih.gov/pmc/" class="csl-entry" role="doc-biblioentry">
PubMed Central Homepage. (2010). In <em>web.archive.org</em>. <a href="https://web.archive.org/web/20101123165045/http://www.ncbi.nlm.nih.gov/pmc/">https://web.archive.org/web/20101123165045/http://www.ncbi.nlm.nih.gov/pmc/</a>
</div>
<div id="ref-https://web.archive.org/web/20160528080726/http://erc.europa.eu/sites/default/files/press_release/files/EuropePMC_press_release_WT_ERC_FINAL.pdf" class="csl-entry" role="doc-biblioentry">
Wayback Machine. (2016). In <em>web.archive.org</em>. <a href="https://web.archive.org/web/20160909054208/https://erc.europa.eu/sites/default/files/press_release/files/EuropePMC_press_release_WT_ERC_FINAL.pdf">https://web.archive.org/web/20160909054208/https://erc.europa.eu/sites/default/files/press_release/files/EuropePMC_press_release_WT_ERC_FINAL.pdf</a>
</div>
</div>
