<p>One of the challenges of collecting metrics for scholarly outputs is persistent identifiers. For journal articles the Digital Object Identifier (DOI) has become the de-facto standard, other popular identifiers are the pmid from PubMed, the identifiers used by Scopus and Web of Science, and the arxiv ID for ArXiV preprints.</p>
<p>For other research outputs the picture is less clear. DOIs are also used for datasets, but so are many other identifiers, in particular in the life sciences.</p>
<p>To collect metrics for research outputs, the requirements are slightly different. We need identifiers understood by the services collecting the metrics, not by the data repository or other service that is holding the research output (the only exception is usage stats, which are generated locally). For many services, in particular social media such as Facebook, Twitter or Reddit, the primary identifier for a resource is a URL. This means that we should have one or more URLs for every research output where we want to track the metrics - typically the publisher or data repository landing page. Since URLs can be messy, Google, Facebook and others have come up with the concept of a <a href="http://googlewebmastercentral.blogspot.de/2009/02/specify-your-canonical.html" data-cite-meta="{}">canonical URL <span class="citation" data-cites="http://googlewebmastercentral.blogspot.de/2009/02/specify-your-canonical.html">(<strong>http://googlewebmastercentral.blogspot.de/2009/02/specify-your-canonical.html?</strong>)</span></a>, and some care should go into constructing proper canonical URLs (see <a href="http://blog.martinfenner.org/2013/10/13/broken-dois/" data-cite-meta="{}">this blog post <span class="citation" data-cites="http://blog.martinfenner.org/2013/10/13/broken-dois/">(<strong>http://blog.martinfenner.org/2013/10/13/broken-dois/?</strong>)</span></a> for examples of what can go wrong).</p>
<p>The <a href="http://www.knowledge-exchange.info/Default.aspx?ID=462" data-cite-meta="{}">Den Haag Manifesto <span class="citation" data-cites="http://www.knowledge-exchange.info/Default.aspx?ID=462">(<strong>http://www.knowledge-exchange.info/Default.aspx?ID=462?</strong>)</span></a> is the result of a <strong><strong>Knowledge Exchange</strong></strong> workshop held in June 2011 and tries to bring Persistent Identifiers and Linked Open Data together. The first principle is very much in line with what I said above:</p>
<blockquote>
Make sure PIDs can be referred to as HTTP URI’s, including support for content negotiation.
</blockquote>
<p>Or, to put this differently: URLs are good enough to start collecting metrics for scholarly outputs. Scientific software is a good example where persistent identifiers are not commonly used (despite efforts such as <a href="https://guides.github.com/activities/citable-code/" data-cite-meta="{}">this one <span class="citation" data-cites="https://guides.github.com/activities/citable-code/">(<strong>https://guides.github.com/activities/citable-code/?</strong>)</span></a>), but we can still collect many meaningful metrics using the repository URL (and the open source software <a href="https://github.com/articlemetrics/lagotto" data-cite-meta="{}">lagotto <span class="citation" data-cites="https://github.com/articlemetrics/lagotto">(<strong>https://github.com/articlemetrics/lagotto?</strong>)</span></a>):</p>
<figure>
<img src="http://blog.martinfenner.org/images/software.lagotto.io.png" class="kg-image" alt="Number of software repositories (out of 1,404) with at least one event. Data from software.lagotto.io (http://software.lagotto.io/?)" /><figcaption aria-hidden="true">Number of software repositories (out of 1,404) with at least one event. Data from <a href="http://software.lagotto.io/" data-cite-meta="{}">software.lagotto.io <span class="citation" data-cites="http://software.lagotto.io/">(<strong>http://software.lagotto.io/?</strong>)</span></a></figcaption>
</figure>
<p>The last three rows are citations in the scholarly literature found via fulltext search of BioMed Central, Europe PMC and PLOS. URLs (in contrast to persistent identifiers represented as strings and/or numbers) are easy to find, the main limitation is not so much using a URL rather than a DOI, but that scientific software typically is mentioned in the text without appearing in the reference list. This makes it hard to impossible to find articles mentioning the software that are not open access, which unfortunately is still the majority of them.</p>
<p>We are of course also tracking the discussion of the software in social media, and are collecting the number of stars and forks in Github and Bitbucket. Overall there is quite a lot of activity, here are some examples:</p>
<ul>
<li><a href="http://software.lagotto.io/works/url/https://github.com/najoshi/sickle" data-cite-meta="{}">Windowed Adaptive Trimming for fastq files using quality <span class="citation" data-cites="http://software.lagotto.io/works/url/https://github.com/najoshi/sickle">(<strong>http://software.lagotto.io/works/url/https://github.com/najoshi/sickle?</strong>)</span></a></li>
<li><a href="https://github.com/lh3/wgsim" data-cite-meta="{}">Reads simulator <span class="citation" data-cites="https://github.com/lh3/wgsim">(<strong>https://github.com/lh3/wgsim?</strong>)</span></a></li>
<li><a href="http://software.lagotto.io/works/url/https://github.com/lh3/seqtk" data-cite-meta="{}">Toolkit for processing sequences in FASTA/Q formats <span class="citation" data-cites="http://software.lagotto.io/works/url/https://github.com/lh3/seqtk">(<strong>http://software.lagotto.io/works/url/https://github.com/lh3/seqtk?</strong>)</span></a></li>
</ul>
<p>All three software repos have been cited in the scholarly literature at least ten times. What is missing is infrastructure that tracks the citations of scientific software, so that we can give proper scientific credit to the authors of the software, and can discover other research projects using the same tools. <a href="http://software.lagotto.io/" data-cite-meta="{}">software.lagotto.io <span class="citation" data-cites="http://software.lagotto.io/">(<strong>http://software.lagotto.io/?</strong>)</span></a> uses a list of software repos collected by Jure Triglav for <a href="http://sciencetoolbox.org/" data-cite-meta="{}">ScienceToolbox <span class="citation" data-cites="http://sciencetoolbox.org/">(<strong>http://sciencetoolbox.org/?</strong>)</span></a>, and a scientific software index is indeed one of the important missing pieces.</p>
<div id="refs" class="references csl-bib-body hanging-indent" data-line-spacing="2" role="doc-bibliography">
<div id="ref-https://web.archive.org/web/20150922174101/http://blogs.plos.org/dx.doi/org/10.1038/nature04478" class="csl-entry" role="doc-biblioentry">
Carmeliet, P. (2005). Angiogenesis in life, disease and medicine. <em>Nature</em>, <em>438</em>(7070), 932–936. <a href="https://doi.org/10.1038/nature04478">https://doi.org/10.1038/nature04478</a>
</div>
<div id="ref-https://web.archive.org/web/20150922174101/http://dx.doi.org/10.1126/science.1160622" class="csl-entry" role="doc-biblioentry">
Contopoulos-Ioannidis, D. G., Alexiou, G. A., Gouvias, T. C., &amp; Ioannidis, J. P. A. (2008). Life Cycle of Translational Research for Medical Interventions. <em>Science</em>, <em>321</em>(5894), 1298–1299. <a href="https://doi.org/10.1126/science.1160622">https://doi.org/10.1126/science.1160622</a>
</div>
<div id="ref-https://web.archive.org/web/20150922174101/http://nobelprize.org/nobel_prizes/medicine/laureates/2005/index.html" class="csl-entry" role="doc-biblioentry">
The Nobel Prize in Physiology or Medicine 2005. (2015). In <em>web.archive.org</em>. <a href="https://web.archive.org/web/20150814032045/http://www.nobelprize.org/nobel_prizes/medicine/laureates/2005/index.html">https://web.archive.org/web/20150814032045/http://www.nobelprize.org/nobel_prizes/medicine/laureates/2005/index.html</a>
</div>
</div>
